ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -129,12 +129,17 @@ public void startAsync() {
       versionChecker.start(KsqlModuleType.SERVER, properties);
     } catch (final Exception e) {
       log.error("Failed to start KSQL Server with query file: " + queriesFile, e);
-      triggerShutdown();
```

Comment:
Was it a bug that this was here previously, since it means triggerShutdown() was being called from both StandaloneExecutor and KsqlServerMain? (Checking my understanding for why this was removed.)

Reply:
I don't think it's a bug necessarily (since triggerShutdown is idempotent). Just not needed.